addons/l10n_nl/migrations/3.3/post-migrate_update_taxes.py

```diff
@@ -64,11 +65,11 @@ def migrate(cr, version):
                 UPDATE account_account_tag_account_tax_repartition_line_rel
                 SET account_account_tag_id = {new_tax_tag_id}
                 WHERE account_tax_repartition_line_id = ANY({repartition_line_ids})
-                AND account_account_tag_id = {old_tax_tag_id}
```

Comment:
Will it not be possible to have conflicts with that (two times the same tag on the same line ; crashing the query) ? :thinking: The old 3.2 script Added the 3BL tag on top of the original 3B tag, but it didn't delete it, did it ? (They probably were kept archived because of the removal of the expression) ; so it's possible that there is both  a 3BL and 3B tag to update on the same line. 

I guess there's a way to delete in an ON CONFLICT clause ? :thinking:

Reply:
Then should I just add the ON CONFLICT clause or instead add on top of the old tags the new one ?